core/rawdb/bor_receipt.go

```diff
@@ -0,0 +1,130 @@
+package rawdb
+
+import (
+	"github.com/maticnetwork/bor/common"
+	"github.com/maticnetwork/bor/core/types"
+	"github.com/maticnetwork/bor/ethdb"
+	"github.com/maticnetwork/bor/log"
+	"github.com/maticnetwork/bor/rlp"
+)
+
+var (
+	borReceiptPrefix = []byte("bor-receipt-") // borReceiptPrefix + number + block hash -> bor block receipt
```

Comment:
What this `number` is going to be ?

Reply:
It's a block number.

Reply:
I think we can make the name `blockNumber`, makes it more intuitive.